src/contact.md

```diff
@@ -2,4 +2,24 @@
 template: contact.html
 title: Contact
 ---
-Internet carrier pigeons will deliver your infos somewhere.
+
+<form method="POST" action="nrrv<))`itkuvtcc(oi)mgjgurgroeFc~gkvjc(eik" id="contact">
```

Comment:
How can we tell where the email is going once obfuscated?
Would it not make sense to use a variable for the email, then encrypt it in the template?